### PR TITLE
Use fuzzy-phrase@phrase-ids

### PR DESCRIPTION
**Pull request by apendleton**
https://github.com/mapbox/fuzzy-phrase/pull/71 introduces a mechanism whereby phrases are assigned auto-incrementing IDs, and information about the IDs of matches is returned by FuzzyPhraseSet methods.

Changes consist of:
- [x] update to fuzzy-phrase@phrase-ids
- [x] make `insert` return a temporary ID for the just-inserted phrase
- [x] make `finish` return a mapping from temporary to final IDs (as a JS ArrayBuffer where index is tmpid and value is final ID)
- [x] make all the `fuzzy_*` methods return `phrase_id_range` keys in all responses
- [x] reflecting a change on the fuzzy-phrase side, require that word replacements be loaded before phrases can be inserted, and error otherwise
- [x] add test that verifies that the tmpid-to-final map is correct
- [x] update all read tests to expect phrase ID ranges

### CODE REVIEW

**Comment by apendleton** [comment]
Similar to https://github.com/mapbox/fuzzy-phrase/pull/71#issuecomment-498080261 -- I added support for getting a phrase given a phrase ID here as well, and published a package. Note that that CI failure above isn't real -- we have Travis set up twice on this repo for some reason, and only one of them has credentials to publish to S3. I can get that sorted out if we care.

**Comment by apendleton** [comment]
I fixed the previous Travis misconfiguration, so now we only build things on travis-ci.com and not also .org, and .com can now successfully publish binaries.

**Comment by apendleton** [comment]
I changed the target of this branch to be a dev branch instead of master, so we can actually review/merge it rather than leaving it in limbo (cc @miccolis if you'll still review, otherwise we'll reassign).

File: native/src/lib.rs [hunk 1]
```diff
@@ -5,7 +5,8 @@ extern crate neon_serde;
 
 use neon::mem::Handle;
 use neon::vm::{This, Lock, FunctionCall, JsResult};
-use neon::js::{JsFunction, Object, JsString, Value, JsUndefined, JsArray, JsBoolean, JsInteger};
```
Comment (aarthykc) [inline native/src/lib.rs]: @apendleton not a blocker for merging this PR but maybe we should open a ticket to upgrade to the new version of neon?

Comment (apendleton) [reply native/src/lib.rs]: I think we'll instead probably deprecate this library altogether and wrap fuzzy-phrase from within carmen-core (under neon 0.3). I'll ticket there and link back.

**Pull request merged**